### DISCUSSION TITLE
Projects: No way to create standalone Markdown cards anymore?

### DISCUSSION THREAD

**Discussion by foosel** [community/community#4422] [Projects and Issues]
I so far was under the impression that the new Projects would be the old Projects plus more amazing features. On that note, is it no longer intended to allow to create first citizen cards not tied to an issue? Case in point, I have a [project board for collecting and tracking long term ideas](https://github.com/orgs/OctoPrint/projects/1). I make heavy use of markdown on the cards there to brainstorm and collect knowledge on the tasks.

With Projects Beta I only seem to be able to create single line cards now, without the possibility to add additional, markdown formatted text. I tried adding a description column, but that also just gets squashed into a single line and not rendered.

Are there any plans to bring back the possibility to have richly formatted cards living next to issues tracked on a bug tracker, or do I have to create a repository specifically just to track long term tasks with formatted info that I can then manage via a Project? I would consider this a step back, I loved that I was able to intersperse free form formatted tasks/cards into my project boards. 

Old:

![image](https://user-images.githubusercontent.com/83657/123945177-298b7e80-d99e-11eb-8bd7-3917a298bf30.png)

New:

![image](https://user-images.githubusercontent.com/83657/123945148-1f698000-d99e-11eb-8dd7-d8d72f3ce397.png)

tldr: What I'd like to see here is for a task to have an additional markdown enabled body. Not rendered in table view, but rendered (possibly collapsed) under the task title in board view. That way I can collect information related to a task that is NOT tied to an issue and still manage it like I could on the old Project boards. On that note I'd also prefer they weren't marked as "draft".

**Comment by jesfer** [comment]
I'm in complete agreement with this. I hope it is just a temporary thing due to "beta". Unformatted oneliners are next to useless to me... A card with a title and markdown content would hit the sweet spot for my "not ready to create an issue yet" needs.

**Comment by MichielDeMey** [comment]
In addition, we could link to certain issues and they would show up as "References". Would be nice to have that back.

<img width="347" alt="Screenshot 2021-09-08 at 14 18 25" src="https://user-images.githubusercontent.com/793406/132507901-a4b9da41-1a50-4ac9-b208-9bfd4b0221e0.png">

**Comment by lyzadanger** [comment]
+1. It's inelegant, but reality dictates that sometimes we just need to add unstructured content to our project boards. Sometimes these get migrated to issues, most often they're just useful reminders or, well, notes. They don't need to do anything fancy or new; we just need a container to stick arbitrary text content into (ideally markdown). Not everything can normalize into an issue straightaway.

**Comment by szul** [comment]
Agreed. We use notes for informal tasks, ideas, and brainstorming before they become official business requirements. Not having these rich notes reduces some of our effectiveness.

**Comment by northwesttreesgaming** [comment]
I mainly use the old version with styles to present to people planned features for my modification for games. This can make it easier to present to the general public quickly and with out little effort, Also I don't think the wiki pages support check boxes so having to do lists are not really possible on the wiki pages unless they plan to add support there for them.
  
And often using issues with style codes just to add notes can create a false issue count of any bug reports so you would be constantly checking if there was new issue reported, this becomes a problem on larger projects with may people reporting issues they find. So really the boards where the best option for a way that can present road maps to people.
  
However I would be open to the idea of a better system in place if GitHub adds a way for roadmaps and notes other than boards and issue, if we can have the same styling stuff as before but on a easier to present page like wiki but more modular like boards this could really help with planning projects and even presenting it to communities.

**Comment by cportner** [comment]
It would also be nice if the old version of writing notes was added back. Right now it seems like only the title is added, forcing me to add the card title, then open it and add the additional information. This is clearly a step back from the old functionality.

**Comment by AdrienVerticalCity** [comment]
It looks like GitHub intends to bring this feature over to beta. Check out the following issue and their public roadmap where it is in the "Future" column:

* https://github.com/github/roadmap/issues/287
* https://github.com/orgs/github/projects/4247

**Reply by krisu5** [reply]
It was removed from todo list after Feb 9, 2022 edit.

And it's not in the other "projects" related roadmaps.

**Comment by Katsute** [comment]
This feature was removed from the roadmap, is this not being added to the new project tables? A single unformatted line simply cannot convey the same information as a markdown card.

**Reply by brian-hayward** [reply]
Agreed, we would also appreciate being able to add this to our new project boards. We like to include brief instructional information about how columns should and should not be used. 

As an alternative, we can create an issue that always sits at the top of a column, but there are obvious cons to that and we've chosen against it.

**Comment by krisu5** [comment]
Without this feature, I will not begin using the new projects.

It was the number one thing I used in classic projects (even with some flaws) and "drafts" won't cut it.

In my view, projects and issues are seperate things. And in my repos, using projects for listing issues feels reduntant.

I rather move to Notion or Trello then start using this. It is a shame, because then less likely people will know that's in the roadmap.

**Comment by phifuh** [comment]
Agree, richly formatted multiline cards living next to issues are a must have. 
This opens up for customisation and is adding things outside the normal "issue´s" scope.
It´s kinda a dealbreaker to not have it in the new projectboards
